src/controllers/ChatController.ts

```diff
@@ -0,0 +1,124 @@
+import ChatAPI, {addUserToChatData, createChatData, deleteUserFromChatData} from "../api/ChatAPI";
+import store from "../modules/Store/Store";
+import {IMessagesItem} from "../components/messagesItem";
+import {ChatSocket} from "../modules/ChatSocked/ChatSocked";
+
+
+
+
+class ChatController {
+
+    socket: ChatSocket | undefined
+
+    private api: ChatAPI;
+
+    constructor() {
+        this.api = new ChatAPI();
+    }
+
+    async getChats() {
+        const chatsResponse = await this.api.getChats();
+
+        const chats: Array<IMessagesItem> = [];
+
+        chatsResponse.forEach((chat) => {
+
+            chats.push({
+                chatID: chat.id,
+                name: chat.title,
+                avatar: {
+                    src: chat.avatar
+                },
+                message: {
+                    preview_text: chat.last_message,
+                    notification: {
+                        count: chat.unread_count,
+                    }
+                }
+            })
+        })
+
+        store.set('chats', {
+            items: chats
+        });
+    }
+
+    async createChat(data: createChatData) {
+
+        try {
+            await this.api.createChat(data);
+
+            await this.getChats();
+
+
+
+
+
+        } catch (e) {
+            throw new Error('createChatController' + e)
```

Comment:
Ну вот, опять же где-то есть, где-то нет)
И лишние отступы стоит поубирать тоже
```suggestion
        try {
            await this.api.createChat(data);

            await this.getChats();
        } catch (e) {
            throw new Error('createChatController' + e)
```